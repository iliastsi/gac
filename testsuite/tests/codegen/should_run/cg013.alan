test_byte () : proc

   correct () : proc
   {
      writeString("\n---\n");
      writeString("10100\n");
   }

   lala (a : byte, b : reference byte) : byte
   {
      return a * b;
   }

   a : byte;
   b : byte;
{
   a = '\x01';
   b = '\0';
   writeByte('\x01');
   writeByte('\0');
   writeByte(a);
   writeByte(b);
   writeByte(lala('\0', a));
   correct();
}
