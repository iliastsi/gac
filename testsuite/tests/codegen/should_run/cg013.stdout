10100
---
10100
